packages/framework/src/state/widget.ts

```diff
@@ -102,12 +100,21 @@ export const createBindingAtom = <T = unknown>(
       );
       return [name, value?.values];
     });
+    let mergedInputs = {};
+    if (context) {
+      mergedInputs = {
+        ...(inputs as object),
+        ...context,
+      };
+    } else {
+      mergedInputs = inputs as object;
+    }
```

Comment:
I'm not following why this is needed? Context should override input properties in the merge below?